io/io.cc

```diff
@@ -73,6 +73,22 @@ void AsyncWriteState::OnCb(Result<size_t> res) {
   owner->AsyncWriteSome(cur, arr.end() - cur, [this](Result<size_t> res) { this->OnCb(res); });
 }
 
+// Read some bytes from buf starting at given offset offs and update it.
+Result<size_t> ReadSomeBytes(const iovec* v, uint32_t len, Bytes buf, off_t* offs) {
```

Comment:
please rename v into dest, it's hard to understand from the signature what's the input and what's the output.